### ISSUE TITLE
Consider remove or compress example images from repo?

### ISSUE THREAD

**Issue by wong2** [luin/readability#92]
https://github.com/ruguoapp/readability/tree/master/examples
these two images are relatively large, which makes clone of this repo slow

**Comment by haroldtreen** [comment]
This is true. I'll try to remember this, but if anyone is free in the meantime and wants to reduce the image sizes that would be welcomed.

**Comment by wong2** [comment]
but due to the nature of git, a PR won't be helpful. You have to do something like this: https://help.github.com/articles/removing-sensitive-data-from-a-repository/

**Comment by haroldtreen** [comment]
Good point 👍 . 

I don't have much experiencing wiping things from history, but I'm assuming you can do it with a PR?

**Comment by wong2** [comment]
AFAIK you can't do it with a PR
